### PR TITLE
build: Run integration tests on PRs and on master

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "A module to configure and run Five Bells services for testing.",
   "main": "index.js",
   "scripts": {
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "integration": "ln -s `pwd` node_modules/five-bells-service-manager && integration-loader && integration all"
```
Comment (michielbdejong) [inline package.json]: using `ln -s` here because `npm link` has side-effects on the host machine.

**Comment by michielbdejong** [comment]
.

**Comment by michielbdejong** [comment]
will get rid of unnecessary `ln -s` and make this run the circleci steps the same way as on ilp-kit

**Review by dappelt — APPROVED** [review]

**Comment by michielbdejong** [comment]
Ah, the `ln -s` was necessary, as you can see in https://circleci.com/gh/interledgerjs/five-bells-service-manager/119, putting it back in

**Pull request merged**